docs/instruments/pysatMadrigal.rst

```diff
@@ -0,0 +1,10 @@
+pysatMadrigal
+-------------
+
+`pysatMadrigal <https://github.com/pysat/pysatMadrigal>`_ provides pysat
+support for data available at the
+`Madrigal database. <http://cedar.openmadrigal.org/>`_. See the
```

Comment:
Since I missed a link thought it'd be good to go back over things.
Found a small typo, a double period in the sentence above.
```suggestion
`Madrigal database. <http://cedar.openmadrigal.org/>`_ See the
```